tests/handlers/test_cluster_pf_handler.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from checkQC.handlers.cluster_pf_handler import ClusterPFHandler
+from checkQC.handlers.qc_handler import QCErrorFatal, QCErrorWarning
+
+from tests.test_utils import get_stats_json
+
+class TestClusterPFHandler(unittest.TestCase):
```

Comment:
Since both `set_qc_config` and `map_errors_and_warnings_to_class_names` are used in the other handler class as well, perhaps you could look into setting this up with a `HandlerUnitTest` super class and let the other ones inherit from that? That would leave less code to write later.